govc/namespace/info.go

```diff
@@ -65,27 +74,28 @@ func (cmd *info) Usage() string {
 }
 
 func (cmd *info) Description() string {
-	return `Displays the details of a vSphere Namespace. 
+	return `Displays the details of a vSphere Namespace.
 
 Examples:
   govc namespace.info test-namespace`
 }
 
 func (cmd *info) Run(ctx context.Context, f *flag.FlagSet) error {
-	rc, err := cmd.RestClient()
+	if f.NArg() != 1 {
+		return flag.ErrHelp
+	}
 
+	rc, err := cmd.RestClient()
 	if err != nil {
 		return err
 	}
 
 	nm := namespace.NewManager(rc)
 
 	d, err := nm.GetNamespace(ctx, f.Arg(0))
-
 	if err != nil {
 		return err
 	}
 
-	cmd.JSON = !cmd.All()
```

Comment:
Just FYI - I did the same thing on another PR over a month ago https://github.com/vmware/govmomi/pull/3368
I'm saying this in case you're working on eliminating the "only-JSON-output-supported" pattern :)

Reply:
I remember, but I've yet to use vLCM (directly at least). I use vSphere Namespaces everyday and am happy to replace stuff like below with the new govc commands, thanks!

```bash
# now govc namespace.create
govc session.login -r -X POST /api/vcenter/namespaces/instances \
  <<<'{"cluster": "domain-c9","namespace": "velero"}'

# now govc namespace.ls
govc session.login -r -X GET /api/vcenter/namespaces/instances | jq -r .[].namespace
```